src/Lecture3.hs

```diff
@@ -82,7 +83,25 @@ Tuesday
   would work for **any** enumeration type in Haskell (e.g. 'Bool',
   'Ordering') and not just 'Weekday'?
 -}
-next = error "TODO"
+getTypeValues :: [Weekday]
+getTypeValues = [minBound :: Weekday .. maxBound :: Weekday]
```

Comment:
You don't need specify local types here because Haskell has type inference and it understands that you expect `Weekday` here 👀 
```suggestion
getTypeValues = [minBound .. maxBound]
```

Reply:
Updated